content/administration/maintain/neutralized_databases.rst

```diff
@@ -0,0 +1,36 @@
+=====================
+Neutralized databases
+=====================
+
+What is a neutralized database?
+===============================
```

Comment:
```suggestion
```
The H2 heading isn't necessary; we usually explain what is the topic at hand straight after the H1 heading :)